index.md

```diff
@@ -12,14 +12,14 @@ seo:
 ---
 
 ## Purpose
-RoboJackets, Inc. is a Georgia non-profit corporation that is dedicated to the promotion, education, and advancement of the field of robotics. We support hands-on, super-curricular activities at the Georgia Institute of Technology. For more information about our programs, please visit our main website, [RoboJackets.org](https://robojackets.org).
+RoboJackets, Inc. is a Georgia non-profit 501(c)(7) corporation dedicated to the promotion, education, and advancement of the field of robotics. We support hands-on, super-curricular activities at the Georgia Institute of Technology. For more information about our programs, please visit our main website, [RoboJackets.org](https://robojackets.org).
```

Comment:
It is a little awkward to work in tax-exempt status in this sentence. I was thinking about leaving it out; thoughts?

Reply:
IMO it's fine as-is. Most people think non-profit and tax-exempt are the same and those who don't will probably recognize at least the prefix 501(c).